subcommands/keys/rotate_root.go

```diff
@@ -12,10 +12,11 @@ import (
 
 func init() {
 	cmd.AddCommand(&cobra.Command{
-		Use:   "rotate <offline key archive>",
```

Comment:
Can you keep an obsolete hidden `rotate` command and add a new `rotate-root` ?
Customers might already have some automation in place, and a new release shouldn't unintentionally break it.

Reply:
that's what `Aliases: []string{"rotate"},` does.